### PR TITLE
Update huawei_lte auth config

### PR DESCRIPTION
**Pull request by scop**
## Proposed change

Update authentication config docs per changes in https://github.com/home-assistant/core/pull/49878

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/49878
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/huawei_lte.markdown [hunk 1]
```diff
@@ -91,6 +86,8 @@ To enable the component, add the following lines to your
 # Example configuration.yaml entry
 huawei_lte:
   - url: http://192.168.100.1/
+    username: ROUTER_USERNAME
+    password: ROUTER_PASSWORD
```
Comment (frenck) [inline source/_integrations/huawei_lte.markdown:90]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely do not need. If required, insert a full configuration sample later that covers special setups or alike.

```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/huawei_lte.markdown (hunk 1 above)
Comment (scop) [reply source/_integrations/huawei_lte.markdown:90]: Username and password are now actually required when configuring the integration for the first time, see parent PR.

Comment (frenck) [reply source/_integrations/huawei_lte.markdown:90]: In that case, the schema below needs updating, as it shows they are optional.

Comment (scop) [reply source/_integrations/huawei_lte.markdown:90]: Ah, very much so, will do.

**Review by frenck — APPROVED** [review]
Thanks, @scop 👍

**Pull request merged**